app/src/main/java/io/heckel/ntfy/msg/DownloadAttachmentWorker.kt

```diff
@@ -182,7 +186,7 @@ class DownloadWorker(private val context: Context, params: WorkerParameters) : W
             Repository.AUTO_DOWNLOAD_NEVER -> return true
             Repository.AUTO_DOWNLOAD_ALWAYS -> return false
             else -> {
-                val size = attachment.size ?: return true // Abort if size unknown
+                val size = attachment.size ?: return false // Don't abort if size unknown
                 return size > maxAutoDownloadSize
             }
```

Comment:
This will prevent any huge downloads: https://github.com/wunter8/ntfy-android/blob/a2ae6e4c21c1076e23e6ce313ad87a623fa71a72/app/src/main/java/io/heckel/ntfy/msg/DownloadIconWorker.kt#L94

Reply:
Ahhh yes. Right. Thanks for the refresher.